actions_test.go

```diff
@@ -322,7 +322,7 @@ snapshot:
    required: ["outfile"]
 `,
 		expectedActions: &Actions{map[string]ActionSpec{
-			"snapshot": ActionSpec{
+			"snapshot": {
```

Comment:
This is just a consequence of automatic gofmt simplification - the inner
types are redundant here and may be elided.